### PR TITLE
Add cnho_stables

### PR DESCRIPTION
**Pull request by xiesh1982**

### CODE REVIEW

File: cnho_stables/assetlist.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
+{
+  "$schema": "../assetlist.schema.json",
+  "chain_name": "cnho stables",
```
Comment (JeremyParish69) [inline cnho_stables/assetlist.json:3]: do not use spaces or _ in chainname property. Just "cnhostables" is fine

File: cnho_stables/chain.json [hunk 2]
```diff
@@ -0,0 +1,112 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "cnho stables",
+  "status": "live",
+  "network_type": "mainnet",
+  "website": "https://cnho.io/",
+  "pretty_name": "cnho stables",
```
Comment (JeremyParish69) [inline cnho_stables/chain.json:7]: No capital letters? Not supposed to be something like "CNHO Stables"?

File: cnho_stables/chain.json [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+      "account_page": "https://explorer.cnho.io/account/${accountAddress}"
+    }
+  ],
+  "keywords": [
+    "CNHO"
+  ],
+  "images": [
+    {
+      "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/cnho_stables/images/chain.png",
+
```
Comment (JeremyParish69) [inline cnho_stables/chain.json:106]: Please remove the unused svg

File: cnho_stables/chain.json [hunk 4]
```diff
@@ -0,0 +1,112 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "cnho stables",
+  "status": "live",
+  "network_type": "mainnet",
+  "website": "https://cnho.io/",
+  "pretty_name": "cnho stables",
+  "chain_id": "cnho_stables-1",
+  "bech32_prefix": "cnho",
```
Comment (JeremyParish69) [inline cnho_stables/chain.json:9]: Please register this prefix: https://github.com/satoshilabs/slips/blob/master/slip-0173.md

File: cnho_stables/assetlist.json (hunk 1 above)
Comment (JeremyParish69) [reply cnho_stables/assetlist.json:3]: or even "cnho"

File: cnho_stables/chain.json (hunk 3 above)
Comment (xiesh1982) [reply cnho_stables/chain.json:106]: ok remove laster

File: cnho_stables/assetlist.json (hunk 1 above)
Comment (xiesh1982) [reply cnho_stables/assetlist.json:3]: ok ,use the cnhostables to replace it

File: cnho_stables/chain.json (hunk 2 above)
Comment (xiesh1982) [reply cnho_stables/chain.json:7]: use CNHO Stables is fine?

File: cnho_stables/chain.json (hunk 4 above)
Comment (xiesh1982) [reply cnho_stables/chain.json:9]: i remove the field of bech32_prefix and is ok?

Comment (alashooinc) [reply cnho_stables/chain.json:9]: It is registered.

**Pull request closed without merging**